node/.vscode/launch.json

```diff
@@ -1,23 +1,14 @@
 {
-  // Use IntelliSense to learn about possible attributes.
-  // Hover to view descriptions of existing attributes.
-  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-  "version": "0.2.0",
-  "configurations": [
-
-    {
-      "type": "node",
-      "request": "launch",
-      "name": "Launch functional-tests app",

```

Comment:
I use this launch task all the time. Why are you removing it?

Reply:
Must have accidentally stomped this when I was moving files around and merging them together.  Added it back.